bin/utils.js

```diff
@@ -16,8 +16,6 @@ const CALLBACK_DEBOUNCE_MAXWAIT = parseInt(process.env.CALLBACK_DEBOUNCE_MAXWAIT
 
 const wsReadyStateConnecting = 0
 const wsReadyStateOpen = 1
-const wsReadyStateClosing = 2 // eslint-disable-line
```

Comment:
Can you return these back? I want our repo to be consistent with upstream. Looks like they left these variables here intentionally

Reply:
Done